app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt

```diff
@@ -211,6 +227,9 @@ open class GameActivity : AppCompatActivity(), GameView, Timer.Listener {
 
             putExtra(GAME_CRASH_KEY, hasCrashed)
         }
+
+        //delays last screen
+        Thread.sleep(GAME_OVER_SCREEN_DELAY)
```

Comment:
To delay an intent there is a better way to do it. You should remove this from this pr (as it doesn't much make sense anyway) and we will fix it in a new pr.

Reply:
Ok ok let's do this